apps/ui/lib/services/helpers.ts

```diff
@@ -929,23 +929,39 @@ export const username = (slug: string) => {
 	return slug.replace(/^user-/, '');
 };
 
+/** Given a timeZone string (i.e. `America/New-York` or `Europe/Brussels`) and an optional locale (defualt `en-US`), will return current time at that TZ */
+export const getLocalTimeForTimezone = (
+	timeZone: string,
+	locale?: string,
+): string | null => {
+	if (!timeZone) {
+		return null;
+	}
+	return new Date().toLocaleString(locale ?? 'en-US', { timeZone });
```

Comment:
@aethernet Looking good, but can we just set the first argument to `toLocaleString()` as `undefined`? From what I understand, when `undefined` the function will fall back to the users locale: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

Haven't tested with all browsers, but this is what I got in Chrome (`en-US` locale):
```
let now = new Date();

now.toLocaleString(undefined, { timeZone: 'Europe/London' })
'8/21/2022, 9:55:41 AM'

now.toLocaleString('en-US', { timeZone: 'Europe/London' })
'8/21/2022, 9:55:41 AM'

now.toLocaleString('ja-JP', { timeZone: 'Europe/London' })
'2022/8/21 9:55:41'
```

Reply:
I'll retry that, i though i did and it crashed on me, but maybe it did it differently

Reply:
Works fine, don't know why it didn't the first time.